NBitcoin/Protocol/Node.cs

```diff
@@ -1427,9 +1453,12 @@ public Transaction[] GetMempoolTransactions(CancellationToken cancellationToken
 		public Transaction[] GetMempoolTransactions(uint256[] txIds, CancellationToken cancellationToken = default(CancellationToken))
 		{
 			AssertState(NodeState.HandShaked);
+

```

Comment:
FYI we never use this method so any leak fixing at this point are not for the FN